CustomFeeValidator.cs

```diff
@@ -62,22 +95,43 @@ public static float GetTotalDebtForJobPurposes(DisplayableDebt debt)
         {
             if (!enabled)
                 return debt.GetTotalPrice();
-            ExistingLocoDebt locoDebt = debt as ExistingLocoDebt;
-            if (locoDebt == null)
-                return debt.GetTotalPrice();
 
-            var locoInTrainset = PlayerManager.LastLoco?.trainset?.cars?.Find(car => car.ID == locoDebt.ID);
-            if (locoInTrainset != null)
-            {
-                float fees = debt.GetTotalPriceOfResources(nonConsumableTypes);
-                DebugLog($"locomotive {locoInTrainset.ID} is in player's last trainset. Fees without consumables = {fees}");
-                return fees;
-            }
-            else
-            {
-                DebugLog($"locomotive {debt.ID} not found in last trainset");
-                return debt.GetTotalPrice();
+            switch (selectedFeeType) {
+                case FeeType.ignore_fees_from_train_with_last_loco:
+                    {

```

Comment:
The curly braces here are not needed and in fact add no value. Note that since there is no `break` these cases fall through to the next `case`. Here there is no problem because every path through ends with returning from the function.

I would prefer each of these cases be broken out to a private method.

Reply:
✅ done